diff --git a/node_modules/@terrylinla/react-native-sketch-canvas/android/build.gradle b/node_modules/@terrylinla/react-native-sketch-canvas/android/build.gradle
index 31e44f2..e4f4dec 100644
--- a/node_modules/@terrylinla/react-native-sketch-canvas/android/build.gradle
+++ b/node_modules/@terrylinla/react-native-sketch-canvas/android/build.gradle
@@ -28,5 +28,5 @@ android {
 }
 
 dependencies {
-    provided "com.facebook.react:react-native:+"
+    implementation("com.facebook.react:react-android")
 }
